features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/GraphQLConsole/GraphQLConsole.jsx

```diff
@@ -158,6 +162,29 @@ export default function GraphQLConsole() {
         }
     }
 
+    function grapgQLToPostman(graphQL, URL) {
+        converter.convert({
+            type: 'string',
+            data: graphQL,
+        }, {}, (error, result) => {
+            if (error) {
+                console.log(error);
+            } else {
+                const urlValue = URL.https;
+                const results = result;
+                results.output[0].data.variable[0].value = urlValue;
```

Comment:
Why this is getting from output[0]? any specific reason?

Reply:
one reason I used it as [0] is that it is given in the sample code of the library. Another reason is that we cannot get output without [0] or putting any other number.